hydroflow/src/scheduled/graph.rs

```diff
@@ -235,9 +234,11 @@ impl Hydroflow {
     pub async fn run_async(&mut self) -> Option<!> {
         loop {
             // Run any work which is immediately available.
-            self.run_available();
-            // Only when there is absolutely no work available in any stratum.
-            // Do we yield to wait for more events.
+            while self.run_tick() {
+                // Yield between each tick to receive more events.
+                tokio::task::yield_now().await;
```

Comment:
Will this still consume 100% cpu usage when you have any operator that is outputing data for each tick, like a group_by::<'static>? Except now it will not block the tokio worker thread?

Reply:
Yupp #380